npc/custom/card_seller.txt

```diff
@@ -40,10 +40,22 @@ OnInit:
 		.@mob_db$  = "mob_db";
 		.@item_db$ = "item_db";
 	}
-	freeloop 1;	
-	.@total = query_sql( "SELECT DISTINCT LEFT( `name_english`, 1 ) AS alphabets FROM `"+ .@item_db$ +"` WHERE `type` = 'Card' AND `name_aegis` IN ( SELECT DISTINCT `drop10_item` FROM `"+ .@mob_db$ +"` WHERE `mode_mvp` = 0 ) ORDER BY alphabets;", .@alphabet$ );
+	freeloop 1;
+	
+	.@string$ = "( `name_aegis` IN ( SELECT DISTINCT `drop1_item` FROM `" + .@mob_db$ + "` WHERE `drop1_nosteal` = 1 AND `mode_mvp` IS NULL ) " +
```

Comment:
Why the nosteal condition?

As for MVP I would suggest doing it with `COALESCE(mode_mvp,0) = 0` instead, just in case someone does not only put NULL or 1 into it.

Reply:
The nosteal is a leftover from a old query